packages/handlersjs-http/lib/servers/node/node-http-request-response.handler.ts

```diff
@@ -74,13 +74,27 @@ export class NodeHttpRequestResponseHandler extends NodeHttpStreamsHandler {
 
         const urlObject: URL = new URL(url, `http://${nodeHttpStreams.requestStream.headers.host}`);
 
-        const httpHandlerRequest: HttpHandlerRequest = {
+        let httpHandlerRequest: HttpHandlerRequest = {
           url: urlObject,
           method,
           headers: nodeHttpStreams.requestStream.headers as { [key: string]: string },
         };
 
-        return { request: body !== '' ? Object.assign(httpHandlerRequest, { body }) : httpHandlerRequest };
+        if (body !== '') {
+
+          if (httpHandlerRequest.headers['content-type'] === 'application/json') {
+
+            httpHandlerRequest = { ...httpHandlerRequest, body: JSON.parse(body) };
+
+          } else {
+
+            httpHandlerRequest = { ...httpHandlerRequest, body };
+
+          }
+
+        }
```

Comment:
I would prefer to keep this a constant. I think that would be possible by adding something like 
`body: body !== '' && (httpHandlerRequest.headers['content-type'] === 'application/json' ? JSON.parse(body) : body)`.